shopyo/modules/box__default/admin/tests/test_admin.py

```diff
@@ -7,7 +7,7 @@
 """
 from flask import request
 from flask import url_for
-from modules.box__default.admin.models import Role, User
+from modules.box__default.admin.models import Role, User, role_user_link
 
```

Comment:
Wrong past practise, can you add each import on new line?

from modules.box__default.admin.models import Role
from modules.box__default.admin.models import User
from modules.box__default.admin.models import role_user_link

Reply:
Done.

Reply:
@rehmanis hummm no ... just a clean style for large code bases.